Smooch.postman_collection.json

```diff
@@ -17153,7 +17153,16 @@
 							"name": "List Message Templates",
 							"request": {
 								"method": "GET",
-								"header": [],
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{JWT}}"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
```

Comment:
My postman is setup a little differently, so I had to stick this in manually

Reply:
I don't think you can put stuff in (the output .json) manually - otherwise how do you upload this as part of the collection to the Postman cloud...?